opensearch-api/lib/opensearch/api/actions/security/change_password.rb

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+#
+# Licensed to OpenSearch B.V. under one or more contributor
```

Comment:
There's no such thing as OpenSearch B.V ;) This was an Elastic license grant which is not needed for all new code, remove.